src/test/java/org/folio/rest/impl/PatronResourceImplTest.java

```diff
@@ -1056,14 +1052,14 @@ final void testPostPatronAccountByIdInstanceByInstanceIdHold(boolean tlrState,
 
   static Stream<Arguments> tlrFeatureStates() {
     return Stream.of(
-      Arguments.of(true, false, true, true),
-      Arguments.of(true, false, true, false),
-      Arguments.of(true, false, false, true),
-      Arguments.of(true, false, false, false),
-      Arguments.of(false, false, true, true),
-      Arguments.of(false, false, true, false),
-      Arguments.of(false, false, false, true),
-      Arguments.of(false, false, false, false)
+      Arguments.of( false, true, true),
+      Arguments.of( false, true, false),
+      Arguments.of( false, false, true),
+      Arguments.of( false, false, false),
+      Arguments.of( false, true, true),
+      Arguments.of( false, true, false),
+      Arguments.of( false, false, true),
+      Arguments.of( false, false, false)
```

Comment:
Are there duplicates? Should the first argument be set to true in half of the cases?